learn/tanssi/external-security-providers/symbiotic.md

```diff
@@ -110,20 +112,133 @@ flowchart LR
     tanssi -- Block Production Services--> Networks
     tanssi -- Security--> Networks
     tanssi -- Data Retrievability--> Networks
+
+    class Symbiotic custom-container
 ```
 
-### Slashing and Rewards {: #slashing-rewards }
+### Rewards {: #rewards }
 
-Well-behaved operators and restakers receive rewards for their participation in TANSSI tokens. Reward payments are managed through the vault.
+Well-behaved operators and restakers are rewarded for their participation with TANSSI tokens. The reward process consists of two main phases: [Reward Distribution Phase](#reward-distribution-phase) and [Reward Claiming Phase](#reward-claiming-phase).
 
-The Tanssi protocol also implements veto-slashing to penalize bad actors' misbehavings. These are the actions that cause slashing events:
+#### Reward Distribution Phase {: #reward-distribution-phase }
 
-1. Producing Invalid Blocks (blocks including invalid transactions, for example)
-2. Invalid Validation (double-signing or breaking protocol rules, for example).
-3. Downtime or Unavailability
-4. Consensus Violations
+The reward distribution phase calculates and allocates rewards through five key steps involving operators, restakers, and smart contracts. The steps are:
 
-When a veto-slashing event is triggered, the authorities designated as resolvers by the vault managers can accept or revert this action.
+1. **Reward Calculation** - Tanssi calculates rewards based on the activity of operators and stakers and then creates a [Merkle root](https://en.wikipedia.org/wiki/Merkle_tree). This Merkle root is a cryptographic fingerprint that summarizes the reward allocations, indicating who receives what. Stakers are rewarded according to their stake in each vault
+2. **Cross-Chain Messaging** - reward allocation data is sent using [XCM](https://docs.moonbeam.network/builders/interoperability/xcm/overview/) (Cross-Consensus Messaging), a standardized protocol for blockchain communication. [Snowbridge](https://wiki.polkadot.network/docs/learn-snowbridge) acts as a trustless bridge between Tanssi and Ethereum
```

Comment:
```suggestion
2. **Reward Data Sent via XCM** - reward allocation data is sent using [XCM](/builders/interoperability/xcm/overview/) (Cross-Consensus Messaging), a standardized protocol for blockchain communication. [Snowbridge](https://wiki.polkadot.network/docs/learn-snowbridge) acts as a trustless bridge between Tanssi and Ethereum
```

We're highlighting cross-chain messaging here, but it doesn't mean much to me if I am looking at this for the first time and just trying to understand how rewards are distributed. I think the other valuable part to highlight is what is being sent via XCM